src/libsyntax/diagnostic_list.rs

```diff
@@ -414,4 +414,6 @@ register_diagnostics! {
     E0694, // an unknown tool name found in scoped attributes
     E0703, // invalid ABI
     E0704, // incorrect visibility restriction
+    E0713, // duplicate `rustc_const_fn_force_stable` attributes
+    E0714, // both `rustc_const_fn_force_stable` attribute and `rustc_const_unstable` attribute
```

Comment:
I thought we were getting rid of `rustc_const_fn_force_stable` as a concept?

Reply:
ups, leftover code